src/main/java/housing/Household.java

```diff
@@ -520,6 +648,22 @@ public boolean isInSocialHousing() {
 
     boolean isFirstTimeBuyer() {
         return isFirstTimeBuyer;
+    }    
+      
+    public double getMonthlyTravelCost(Transport transport){
```

Comment:
Yes, the difference between these two methods is that getMonthlyTravelCost check several inputs (including home) are not null. I think this method should be helpful in two aspects below: first, sometimes users may forget to check the inputs if they are null or not. second, it may also help to make the codes more readable, as users do not need to check the inputs every time when they need to use this method.

Reply:
Ok, well, if you think it is worth it, then, of course, leave it as it is, ;-)